### ISSUE TITLE
Fails on non-multilib systems

### ISSUE THREAD

**Issue by alethiophile** [iarna/xenforo-to-epub#1]
Due to some of the provided binaries, this program requires multilib (32-bit runtime libraries) be set up on 64-bit Linux installs. This requirement isn't documented, and the failure message isn't obvious. (Seems to only apply during generate.)

Possible fixes: provide 64-bit binaries (as well? exclusively? does anyone actually own a 32-bit system anymore?), document the requirement, catch the ENOENT and print a meaningful error message

**Comment by iarna** [comment]
Huuh? I had no idea! I'm going to go hunting for C dependencies and remove them if I can, which should solve this issue.

Can you provide a copy of the error message you're seeing?

**Comment by iarna** [comment]
(Hmm, I'm not seeing any `gyp` files, which is what controls compilation of modules, which implies that this shouldn't be possible! =D Hmm, I guess I'll dig up a Linux VM and try from there.)

**Comment by iarna** [comment]
Ah, ok found it…

```
Error: spawn /home/rebecca/.nvm/versions/node/v4.7.0/lib/node_modules/fetch-fic/node_modules/streampub/node_modules/htmltidy/bin/linux/tidy ENOENT
   at exports._errnoException (util.js:907:11)
   at Process.ChildProcess._handle.onexit (internal/child_process.js:189:32)
   at onErrorNT (internal/child_process.js:355:16)
   at nextTickCallbackWith2Args (node.js:458:9)
   at process._tickCallback (node.js:372:17)
```

_grumbles at`htmltidy`_

So yeah, it looks like `htmltidy` just straight up bundles binaries w/o even an option to compile. That's terrible! I'll work on getting rid of that!

**Comment by iarna** [comment]
Ok, this is fixed in 3.2.2 as of the new Streampub (via https://github.com/iarna/streampub/commit/1b060bcdba7d0ec66cefa5a2e228d984ef4a5c75).

Thanks for pointing that out!
